### ISSUE TITLE
alignak-webui failing to run in virtualenv 

### ISSUE THREAD

**Issue by arthurzenika** [Alignak-monitoring-contrib/alignak-webui#274]
```
$ mkvirtualenv alignak
$ pip install alignak-webui
[snip]
$ alignak-webui
Traceback (most recent call last):
  File "/home/arthur/.virtualenvs/alignak/bin/alignak-webui", line 7, in <module>
    from alignak_webui.app import main
  File "/home/arthur/.virtualenvs/alignak/local/lib/python2.7/site-packages/alignak_webui/app.py", line 91, in <module>
    from alignak_webui.webui import WebUI
  File "/home/arthur/.virtualenvs/alignak/local/lib/python2.7/site-packages/alignak_webui/webui.py", line 40, in <module>
    from alignak_webui.utils.plugin import Plugin
  File "/home/arthur/.virtualenvs/alignak/local/lib/python2.7/site-packages/alignak_webui/utils/plugin.py", line 38, in <module>
    from alignak_backend.models import register_models
ImportError: No module named alignak_backend.models
```

**Comment by mohierf** [comment]
The last pip package for the webui has not yet the correct requirements. I fixed the requirements in the repo but not yet published a version with this fix.

You should `pip install alignak-backend` before `pip install alignak-webui`

**Comment by mohierf** [comment]
Fixed with publihed version 0.8.6, please re-open if not
